connectors/accumulo/translator-accumulo/src/test/java/org/teiid/translator/accumulo/TestAccumuloMetadataProcessor.java

```diff
@@ -17,61 +17,80 @@
  */
 package org.teiid.translator.accumulo;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.*;
```

Comment:
our static analysis engine detects that your pull request include change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.